2.9.0/done/cherrypicks.md

```diff
@@ -0,0 +1,48 @@
+# cherry picks
+## bc breaking
+## deprecation
+## new features
+## improvements
+### inductor
+- [Graph Partition] improve custom op output alias ([#163380](https://github.com/pytorch/pytorch/pull/163380))
+
+## bug fixes
+### cuda
+- [cuDNN][SDPA][submodule] Roll-back cuDNN frontend upgrade, update Meta registration ([#163265](https://github.com/pytorch/pytorch/pull/163265))
```

Comment:
remove starting tags, see if this can be combined with the message of another commit. (as it seems to roll something back)